psyneulink/core/llvm/execution.py

```diff
@@ -151,9 +151,9 @@ def cuda_execute(self, variable):
 
 class FuncExecution(CUDAExecution):
 
-    def __init__(self, component, execution_ids=[None]):
+    def __init__(self, component, execution_ids=[None], tags = frozenset()):
```

Comment:
`tags` should be compulsory keyword arg

arg assignment shouldn't be surrounded by whitespace

Reply:
this change would be inconsistent with the existing paradigm for `CompExecution`:

`class CompExecution(CUDAExecution):
    def __init__(self, composition, execution_ids=[None], additional_tags=frozenset()):`

not sure if we wanted to change that constructor as well

Reply:
I'd say that `additional_tags` should follow the same pattern as `tags`, but that can be done separately.

Reply:
https://github.com/PrincetonUniversity/PsyNeuLink/issues/1695

made an issue for it

Reply:
the issue doesn't make much sense to me. `tags` argument should be added when/as needed.
maybe I phrased it wrong; 'compulsory keyword arg` == `tags` should be keywords-only argument when added.

Reply:
ah, i see what you meant. alright, will modify the issue to reflect this.

Reply:
thanks. it also applies to all other `tags` arguments. it should always be keyword-only.